### PR TITLE
Update conversant adapter for new prebid-server interface

### PR DESCRIPTION
**Pull request by johnwier**
Update the conversant adapter to use the new prebid-server interface

### CODE REVIEW

File: adapters/conversant/conversant.go [hunk 1]
```diff
@@ -1,291 +1,177 @@
...
 
-			if len(params.API) > 0 {
-				imp.Video.API = make([]openrtb.APIFramework, 0, len(params.API))
-				for _, api := range params.API {
-					imp.Video.API = append(imp.Video.API, openrtb.APIFramework(api))
-				}
-			}
+func parseCnvrParams(imp *openrtb.Imp, cnvrExt openrtb_ext.ExtImpConversant) {
+	imp.DisplayManager = "prebid-s2s"
+	imp.DisplayManagerVer = "1.0.1"
```
Comment (pycnvr) [inline adapters/conversant/conversant.go]: Let's use prebid-server's version number here.

File: adapters/conversant/conversant.go [hunk 2]
```diff
@@ -1,291 +1,177 @@
...
+	imp.BidFloor = cnvrExt.BidFloor
+	imp.TagID = cnvrExt.TagID
 
-			// Include protocols, mimes, and max duration if specified
-			// These properties can also be specified in ad unit's video object,
-			// but are overridden if the custom params object also contains them.
+	// Take care not to override the global secure flag
+	if (imp.Secure == nil || *imp.Secure == 0) && cnvrExt.Secure != nil {
+		imp.Secure = cnvrExt.Secure
```
Comment (pycnvr) [inline adapters/conversant/conversant.go:84]: I think this should always be secure now?  What do you think?

File: openrtb_ext/imp_conversant.go [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package openrtb_ext
+
+type ExtImpConversant struct {
+	SiteID      string   `json:"site_id"`
+	Secure      *int8    `json:"secure"`
+	TagID       string   `json:"tag_id"`
+	Position    *int8    `json:"position"`
+	BidFloor    float64  `json:"bidfloor"`
+	Mobile      *int8    `json:"mobile"`
```
Comment (pycnvr) [inline openrtb_ext/imp_conversant.go]: Mobile seemed to be unused?

**Review by pycnvr — APPROVED** [review]

File: adapters/conversant/conversant.go (hunk 2 above)
Comment (johnwier) [reply adapters/conversant/conversant.go:84]: changing the default behavior is concerning, I pulled a few random requests from kibana and most seemed to set it already, but I assume the ones that aren't are defaulting to 0

File: openrtb_ext/imp_conversant.go (hunk 3 above)
Comment (johnwier) [reply openrtb_ext/imp_conversant.go]: I can remove that from the adapter and this file.  anywhere else it should be removed from

**Comment by johnwier** [comment]
finished internal review

**Pull request closed without merging**